src/JsonLd/Serializer/ItemNormalizer.php

```diff
@@ -60,7 +60,7 @@ public function supportsNormalization($data, $format = null)
      */
     public function normalize($object, $format = null, array $context = [])
     {
-        $resourceClass = $this->resourceClassResolver->getResourceClass($object, $context['resource_class'] ?? null, true);
+        $resourceClass = $this->resourceClassResolver->getResourceClass($object, get_class($object), true);
```

Comment:
Oh, yes I see.
Then I guess it is sufficient to pass null as resourceClass parameter